types/types.go

```diff
@@ -108,3 +108,11 @@ func (u *User) GetUsername() string {
 func (u *User) GetEmail() string {
 	return u.Email
 }
+
+type UploadStatus int
+
+const (
+	StatusUploadFailed    UploadStatus = -1
+	StatusUploadPending   UploadStatus = 0
+	StatusUploadCompleted UploadStatus = 1
```

Comment:
Naming is very ambiguous.
we know that it is a status, so the names should be:

UploadFailed
UploadPending
UploadCompleted